### DISCUSSION TITLE
How do you undelete a password from the Trash?

### DISCUSSION THREAD

**Discussion by Snake883** [keeweb/keeweb#2175] [Q&A / Support]
I accidentally deleted a password (thought I was only deleting the attachment).
I can see the password in the Trash.
I cannot find a way to undelete, or undo a delete.

**Comment by Aetherinox** [comment]
If you go to your trash, hold your mouse down on the left side of the entry, and drag it back to your vault. 

Should probably add a "Restore from Trash" option. I didn't see one (new to development).

![cT04DIkTO9](https://github.com/keeweb/keeweb/assets/118329232/b8c83869-eaaf-427c-bb29-1c4e95e835ab)
